### PR TITLE
Deny Changes

### PR DESCRIPTION
**Pull request by JTAfo1**
Check this out

### CODE REVIEW

File: JTAwashere.txt [hunk 1]
```diff
@@ -1 +1,2 @@
 HEEEEERRRREEEESSSS JOSHIIEEEEE!!!!!!
+I will spell your name wrong Zober
```
Comment (ZuberMah) [inline JTAwashere.txt]: My name is spelt zuber

**Review by ZuberMah — CHANGES_REQUESTED** [review]
spell name correctly

File: JTAwashere.txt (hunk 1 above)
Comment (JTAfo1) [reply JTAwashere.txt]: Alright will do

Comment (JTAfo1) [reply JTAwashere.txt]: Is that okay now ?

**Review by ZuberMah — APPROVED** [review]

**Pull request merged**